src/bundle/Resources/views/layout.html.twig

```diff
@@ -4,6 +4,7 @@
         <meta charset="UTF-8" />
         <meta name="CSRF-Token" content="{{ csrf_token('authenticate') }}" />
         <meta name="SiteAccess" content="{{ app.request.get('siteaccess').name }}" />
+        <meta name="LanguageCode" content="{{ language.languageCode }}" />
```

Comment:
This supposed to be a languageCode of editing content. I don't think this variable is available across whole admin system. It should be scope only for Page Builder edit.